README.md

```diff
@@ -80,6 +80,16 @@ Run `sensei --help` for available commands and options.
 - Run `sensei pdf`
 - PDFs are generated inside `pdf` folder
 
+⚠️ If running native sensei on Mac M1, you need to follow special steps:
```

Comment:
Great!
But I would put it in the paragraph "Installation > Using Node.js", as it's installation, not usage.

Reply:
Thanks for the review!

Done ✅